config/nmap/nmap.inc

```diff
@@ -71,9 +67,11 @@ function nmap_custom_php_validation_command($post, & $input_errors) {
 function nmap_custom_add_php_command() {
 	$nmap_options = "";
 
-	if (function_exists("is_ipaddrv6") && function_exists("is_subnetv6"))
-		if (is_ipaddrv6($_POST['hostname']) || is_subnetv6($_POST['hostname']))
+	if (function_exists("is_ipaddrv6") && function_exists("is_subnetv6")) {
```

Comment:
You can safely remove these checks, those functions are part of pfSense for a long time